docs/resources/User.md

```diff
@@ -59,6 +60,12 @@ There are other rules and restrictions not shared here for the sake of spam and
   "accent_color": 16711680,
   "premium_type": 1,
   "public_flags": 64
+  "clan": {
+    "identity_guild_id": "1234647491267808778",
+    "identity_enabled": true,
```

Comment:
if they've disabled identity shouldn't this whole clan object just not be serialized?